.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+dist: trusty
+
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=1.x-dev
+
+matrix:
+  include:
+    - php: 5.6
+      env: DB=MYSQL PHPUNIT_TEST=core,admin
+    - php: 7.0
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 7.1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+    - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1
+
+before_script:
+  # Init PHP
+  - phpenv rehash
+  - phpenv config-rm xdebug.ini
+
+  # Install composer dependencies
+  - composer validate
+  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:2.0.x-dev; fi
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+
+script:
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit --testsuite ${PHPUNIT_TEST/1/Default}; fi
```

Comment:
I still don't really understand this format. We don't need to run coverage or phpcs tests, so we can skip a few things in this now. Example: https://github.com/creative-commoners/recipe-collaboration/blob/pulls/1.0/initialize/.travis.yml

Reply:
True there are unnecessary bits in there, I'll change that.